src/pages/index.js

```diff
@@ -2,11 +2,13 @@ import Head from "next/head";
 import Image from "next/image";
 import { Inter } from "next/font/google";
 import styles from "@/styles/Home.module.css";
-import FooterBar from "@/common/FooterBar";
 import ReusableNavLink from "@/common/ReusableNavLink";
+import CustomizableAside from "@/components/customizableAside"
+import LoginForm from "@/Login/login"
```

Comment:
This `LoginForm` import did not render correctly on my local app 😟  Please double-check.

Reply:
Oh, I made a mistake ma, I used the capital letter L, it should be:
Import LoginForm from "/login/login"

Reply:
For simplicity make every route e.g login be just login

and rather than saying login.js just index.js to let Nextjs use just the folder as the main route

So `import LoginForm from "@/login"`